src/gui/mzroll/projectsaveworker.cpp

```diff
@@ -89,7 +89,9 @@ void TempProjectSaveWorker::saveProject(const bool saveRawData)
     if (_currentProjectName.isEmpty()) {
         auto now = QDateTime::currentDateTime();
         auto tempFileName = now.toString("dd_MM_yyyy_hh_mm_ss") + ".emDB";
-        auto firstSampleFile = _mw->getSamples()[0]->fileName;
+        string firstSampleFile = "";
+        if (!_mw->getSamples().empty())
+            auto firstSampleFile = _mw->getSamples()[0]->fileName;
```

Comment:
The good/bad button should not even trigger an "autosave" kind of event, when there is no peak-group selected in a table.

Reply:
Okkay.. will make the change.